Jenkinsfile

```diff
@@ -15,14 +15,14 @@ pipeline {
 	stages {
 		stage('Build') {
 			steps {
-				wrap([$class: 'Xvnc', useXauthority: true]) {
+				xvnc(useXauthority: true) {
 					sh """
-						mvn clean verify --batch-mode -Dmaven.repo.local=$WORKSPACE/.m2/repository \
-						-Pbree-libs \
-						-Papi-check \
-						-Pjavadoc \
+					mvn clean verify -Dmaven.repo.local=$WORKSPACE/.m2/repository \
+						--fail-at-end --update-snapshots --batch-mode --no-transfer-progress --show-version --errors \
+						-Pbree-libs -Papi-check -Pjavadoc -Ptck \
 						${env.BRANCH_NAME=='master' ? '-Peclipse-sign': ''} \
-						-Dmaven.test.error.ignore=true -Dmaven.test.failure.ignore=true \
```

Comment:
Neither maven-surefire nor Tycho use the property `maven.test.error.ignore`, they only use `maven.test.failure.ignore`: 
- https://github.com/search?q=repo%3Aapache%2Fmaven-surefire+maven.test.error.ignore&type=code
- https://github.com/search?q=repo%3Aeclipse-tycho%2Ftycho+maven.test.error.ignore&type=code

It's often set within SDK projects, but I have the impression that this property actually does not have an effect.
@laeubi or do you now something else that considers it?